decidim-core/spec/system/homepage_spec.rb

```diff
@@ -269,15 +265,11 @@
           it "shows the metrics block empty" do
             within "#metrics" do
               expect(page).to have_content("Participation in figures")
-              within ".metric-charts:first-child" do
-                Decidim.metrics_registry.highlighted.each do |metric_registry|
-                  expect(page).to have_no_css(%(##{metric_registry.metric_name}_chart))
-                end
+              Decidim.metrics_registry.highlighted.each do |metric_registry|
+                expect(page).to have_no_css(%(##{metric_registry.metric_name}_chart))
```

Comment:
> `%(##{metric_registry.metric_name}_chart)`

Isn't this very complicated? `"##{metric_registry.metric_name}_chart" maybe?

Reply:
Ok